src/apps/backend/modules/account/internal/store/account_repository.py

```diff
@@ -1,13 +1,48 @@
 from pymongo.collection import Collection
-
+from pymongo.errors import OperationFailure
 from modules.account.internal.store.account_model import AccountModel
 from modules.application.repository import ApplicationRepository
 
+ACCOUNT_VALIDATION_SCHEMA = {
+    "$jsonSchema": {
+        "bsonType": "object",
+        "required": ["username", "hashed_password"],
```

Comment:
With phone based login, `hashed_password` does not look like it should be requried.